src/components/Admin/Messenger.jsx

```diff
@@ -7,90 +7,76 @@ import Input from "./Input";
 import Textarea from "./Textarea";
 import Button from "./Button";
 import toast from "react-hot-toast";
-
-const ADDRESSEES = [
-  {
-    uid: "1",
-    name: "Menthy Wu",
-    email: "yhung022@ucr.edu",
-    team: "b",
-    major: "Computer Science",
-    status: "pending",
-    selected: false,
-  },
-  {
-    uid: "2",
-    name: "Divyank Shah",
-    email: "yhung022@ucr.edu",
-    team: "c",
-    major: "Computer Science",
-    status: "accepted",
-    selected: false,
-  },
-  {
-    uid: "3",
-    name: "Shing Hung",
-    email: "yhung022@ucr.edu",
-    team: "d",
-    major: "Computer Science",
-    status: "rejected",
-    selected: false,
-  },
-];
+import Upload from "./Upload";
+import { userList } from "@/data/mock/messengerUsers";
+import { messengerFilters } from "@/data/Filters";
 
 const Messenger = () => {
-  const [subjectText, setSubjectText] = useState("");
-  const [messageBody, setMessageBody] = useState("");
-  const [filters, setFilters] = useState({
-    accepted: false,
-    rejected: false,
-    pending: false,
-    volunteers: false,
-    mentors: false,
-    judges: false,
+  const [email, setEmail] = useState({
+    sendto: [],
+    subject: "",
+    body: "",
+    files: [],
   });
-  const [filteredParticipants, setfilteredParticipants] = useState(ADDRESSEES);
+  const [filters, setFilters] = useState(messengerFilters);
+  const [user, setUser] = useState(userList);
   const clickHandler = () => {
-    if (subjectText === "") {
+    if (email.subject === "") {
       toast("❌ Please add a subject!");
       return;
     }
-    if (messageBody === "") {
+    if (email.body === "") {
       toast("❌ Please add a body!");
       return;
     }
-    const emails = filteredParticipants.map((user) => user.email);
-    console.log({
-      sendto: emails,
-      subject: subjectText,
-      body: messageBody,
+    const emails = user.filter((user) => {
```

Comment:
don't need this, assume that u will get an emails array directly, we will need to make a backend call anyways otherwise we have to grab all the user information which is not needed